common/src/main/java/com/wynntils/models/items/WynnItemCache.java

```diff
@@ -14,6 +14,7 @@ public class WynnItemCache {
     public static final String TOOLTIP_KEY = "tooltip";
     public static final String SEARCHED_KEY = "searched";
     public static final String FAVORITE_KEY = "favorite";
+    public static final String EMERALD_PRICE_KEY = "emerald_price";
```

Comment:
Maybe just PRICE = "price"? The emerald prefix feels a bit redundant, and a single word makes the key name fit better with the rest.